src/python/WMCore/WorkQueue/Policy/Start/Block.py

```diff
@@ -227,8 +227,13 @@ def getMaskedBlocks(self, task, dbs, datasetPath):
             
             for slumis in grouper(lumis, 1000):
                 for block in blocks:
-                    slicedFiles = dbs.dbs.listFileArray(block_name=block.get('block_name'), run_num=run,
-                                                        lumi_list=slumis, detail=True)
+                    # run_num=1 will not be allowed anymore in this API (only for LFNs, not blocks
+                    # nor dataset), starting on Feb/2016
+                    if run==1:
+                        slicedFiles = dbs.dbs.listFileArray(block_name=block.get('block_name'), lumi_list=slumis, detail=True)
```

Comment:
Hi Alan, Yuyi told me this call wouldn't work. when lumi_list is specified, run number has to be specified.
How about something like following?
https://github.com/dmwm/WMCore/compare/master...ticoann:dbs_api_change?expand=1